src/Libs/Soa/v1/Yousign.php

```diff
@@ -411,6 +412,32 @@ public function deleteSignatureRequest(string $signatureRequestId, CancelSignatu
         return new SignatureRequestResponse($response->json());
     }
 
+    /**
+     * @param string $signatureRequestId
+     *
+     * @return Response
+     */
+    public function deleteSignatureRequest(string $signatureRequestId): Response
+    {
+        /** @var string $url */
+        $url = implode(
+            self::URL_SEPARATOR,
+            [
+                self::SIGNATURE_REQUESTS_BASE_URL,
+                $signatureRequestId,
+            ],
+        );
+
+        /** @var Response $response */
+        $response = $this->apiClient->delete($url);
+
+        if ($response->status() !== ResponseHttp::HTTP_NO_CONTENT) {
```

Comment:
La tua preoccupazione di controllare lo status code qui, esattamente a cosa è finalizzata? Perché se la chiamata alla 432 non ha dato esito di successo, a questa riga non ci si arriva.

in caso hai `$response->noContent()`